apps/client/src/app/pages/portfolio/allocations/allocations-page.html

```diff
@@ -5,6 +5,30 @@ <h3 class="d-flex justify-content-center mb-3" i18n>Allocations</h3>
     </div>
   </div>
   <div class="proportion-charts row">
+    <div class="col-md-6">
+      <mat-card class="mb-3">
+        <mat-card-header class="w-100">
+          <mat-card-title i18n>By Symbol</mat-card-title>
+          <gf-toggle
+            [defaultValue]="period"
+            [isLoading]="false"
+            [options]="periodOptions"
+            (change)="onChangePeriod($event.value)"
+          ></gf-toggle>
+        </mat-card-header>
+        <mat-card-content>
+          <gf-portfolio-proportion-chart
+            [baseCurrency]="user?.settings?.baseCurrency"
+            [isInPercent]="false"
```

Comment:
I guess this should be `[isInPercent]="hasImpersonationId"` because you won't have absolute values in impersonation mode.